### ISSUE TITLE
Not working with Things 3.17.7 and onwards

### ISSUE THREAD

**Issue by emacsbliss** [thingsapi/things-cli#17]
Looks like Things has changed its database location starting at some version(not sure exactly which version it starts) but I can observe it starting version 3.17.7.

database location used to be at:
```
/Users/emacsbliss/Library/Group Containers/JLMPQHK86H.com.culturedcode.ThingsMac/Things Database.thingsdatabase/main.sqlite
```

but now it's at:
```
/Users/emacsbliss/Library/Group Containers/JLMPQHK86H.com.culturedcode.ThingsMac/ThingsData-B4WGR/Things Database.thingsdatabase/main.sqlite
```

I can work around by export environment variable `THINGSDB`.

But after that I hit another error when running:

```
things-cli --json today
```

The error is:
```
sqlite3.OperationalError: no such column: TASK.dueDate
```

So I guess they must have changed the database schema as well..

can we update the code to work with latest version of the Things?

Thanks a lot!

**Comment by AlexanderWillner** [comment]
I think this is already fixed in release 0.2.1. (may this year) - which version are you using?

**Comment by emacsbliss** [comment]
sorry, my bad, didn't realize I'm using an old version (0.1.3).

Latest version works.
